new/LIT6914NuzhatAsrar.xml

```diff
@@ -7,7 +7,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <titleStmt>
                 <title xml:lang="ar" xml:id="t1">نزهة الاسرار وطهارة الاقذار</title>
                 <title xml:lang="ar" type="normalized" corresp="#t1" >Nuzhat al-asrār wa-ṭahārat al-aqḏār</title>
```

Comment:
I believe that according to our guidelines, it should be al-ʾasrār and al-ʾaqḏār